tests/e2e/utils/src/flows/merchant.js

```diff
@@ -210,11 +211,53 @@ const merchant = {
 		} );
 	},
 
-  openImportProducts: async () => {
+  	openImportProducts: async () => {
 		await page.goto( WP_ADMIN_IMPORT_PRODUCTS , {
 			waitUntil: 'networkidle0',
 		} );
 	},
+
+	openWordPressUpdatesPage: async () => {
+		await page.goto( WP_ADMIN_WP_UPDATES, {
+			waitUntil: 'networkidle0',
+		} );
+	},
+
+	installAllUpdates: async ( updateWordPress = true, updatePlugins = true, updateThemes = true ) => {
```

Comment:
I think it would be less confusing to downstream users to have three functions: `updateWordPress`,`updatePlugins`, and `updateThemes`. Then installAll would not need parameters and could call the other three functions.

Reply:
I like that suggestion. I'll go ahead and break these up.